Examples/Examples/FloatingPanelExampleView.swift

```diff
@@ -38,7 +41,15 @@ struct FloatingPanelExampleView: View {
             map: dataModel.map,
             viewpoint: initialViewpoint
         )
-        .floatingPanel(selectedDetent: $selectedDetent, isPresented: isPresented) {
+        .onAttributionBarHeightChanged {
+            attributionBarHeight = $0
+        }
+        .ignoresSafeArea(.keyboard, edges: .bottom)
+        .floatingPanel(
+            attributionBarHeight: attributionBarHeight,
```

Comment:
By passing the attribution bar height to the floating panel we can prevent the overlap and center the panel right in the geo view.

Reply:
Before:
<img width="354" alt="Screenshot 2023-10-26 at 10 55 55" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/988cdc23-4d67-4715-8832-e49a9db70b9d">
After:
<img width="378" alt="Screenshot 2023-10-26 at 10 52 35" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/5b42dc69-4832-4d38-b9b0-b53fb3f66e18">